test_requests.py

```diff
@@ -411,6 +416,21 @@ def test_DIGESTAUTH_QUOTES_QOP_VALUE(self):
         r = requests.get(url, auth=auth)
         assert '"auth"' in r.request.headers['Authorization']
 
+    def test_DIGESTAUTH_THREADED(self):
```

Comment:
I hate this style of function naming. I appreciate there are other tests in here that do it, but let's not use the fact that some other tests slipped past me to make a mistake here. Can we return this to being `snake_case` please?